Changelog.md

```diff
@@ -5,6 +5,9 @@
 
     - Removes the compatibility of this with `ReSwift-Recorder`, which itself is being deprecated.
     - Ensures compatibility with latest versions of `ReSwift` which have removed these types.
+- Renamed argument labels for modern Swift conformance ([#116](<https://github.com/ReSwift/ReSwift-Router/pull/115>)) - @TaeJoongYoon
```

Comment:
```suggestion
- Renamed argument labels for modern Swift conformance ([#116](<https://github.com/ReSwift/ReSwift-Router/pull/116>)) - @TaeJoongYoon
```

(fixed the pull request number in the URL)

Reply:
My apology for the typo...

Reply:
Do I have to do anything extra?

Reply:
I don't see anything else. I'll merge it once this is fixed 👍

Reply:
Okay. I fixed it :)